go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/mjibson/go-dsp
+
+go 1.21.1
```

Comment:
Adding this file would definitely need to be in a separate PR. And since this repo doesn't have any tests, I'm not inclined to do that because I'm worried about unknown breakage. Also it should be the lowest supported go version, not the highest. In summary this change would need:

- separate PR
- running `go test` in a github action
- minimum supported go version